### DISCUSSION TITLE
iptables errors on newest version

### DISCUSSION THREAD

**Discussion by jerrodlv** [bubuntux/nordvpn#395] [General]
Hey all - recently updated my nord container after i noticed that it hasn't been working. Likely due to old version using user/pass, etc...

SO i generated a Nord token, changed my compose over to use that, and added 
cap_add:
      - NET_ADMIN
      - NET_RAW

But now when i start the container i get a bunch of these errors:
`iptables v1.8.7 (nf_tables): Could not fetch rule set generation id: Invalid argument`

and finally at the end

```
FATAL: iptables is not functional. Ensure your container config adds --cap-add=NET_ADMIN,NET_RAW
--- Due to errors, routing has been disabled ---
```

Unable to connect to qbit after this is running. Has anyone else seen these issues? Maybe some iptables version issue?

For reference, running docker and compose on Synology

**Comment by mhhplumber** [comment]
Nord officially decommed user/pass login a week or two ago.

The iptables issue I am guessing is because on the latest merge I updated to a newer ubuntu version. Hoping a version bump fixes. I've seen a few users with issues mostly synology, 1 or 2 qnap so been working on getting a virtualized env to try to reproduce the error on. I'll spend some time on this the next few days.

**Comment by sagar-aps** [comment]
I'm getting a similar error while using TOKEN instead of USER and PASS. 
```
fix-attrs: warning: fix-attrs is deprecated, please fix volume permissions in your container manager instead
iptables v1.8.7 (legacy): can't initialize iptables table `filter': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
```
